openreview/journal/assignment.py

```diff
@@ -0,0 +1,117 @@
+from .. import openreview
+from .. import tools
+
+from openreview.api import Edge
+
+import random
+from tqdm import tqdm
+
+class Assignment(object):
+
+    def __init__(self, journal):
+        self.client = journal.client
+        self.journal = journal
+
+
+    def setup_ae_assignment(self, note):
+        venue_id=self.journal.venue_id
+        action_editors_id=self.journal.get_action_editors_id()
+        authors_id=self.journal.get_authors_id(number=note.number)
+
+        action_editors = self.journal.get_action_editors()
+        action_editor_profiles = tools.get_profiles(self.client, action_editors, with_publications=True)
+
+        authors = self.journal.get_authors(number=note.number)
+        author_profiles = tools.get_profiles(self.client, authors, with_publications=True)
+
+        ## Create conflict and affinity score edges
+        for ae in action_editors:
+
+            edge = Edge(invitation = self.journal.get_ae_affinity_score_id(),
+                readers = [venue_id, authors_id, ae],
+                writers = [venue_id],
+                signatures = [venue_id],
+                head = note.id,
+                tail = ae,
+                weight=round(random.random(), 2)
```

Comment:
why do we assign a random affinity score here, is this a placeholder for a future affinity score computation?

Reply:
yes, I'm waiting for the expertise API to be ready to do the integration.